src/assets/lang/en.json

```diff
@@ -1,6 +1,10 @@
 {
   "header": "OpenFisca makes law computable. It is a free and open-source computation engine that allows modelling tax and benefit systems in a collaborative and transparent way. This software is used by researchers and public agencies.",
-  "stats": "{variablesCount, number} variables and {parametersCount, number} parameters referenced to modelize the french tax-benefit system.",
+  "learnMore":"Learn more.",
+  "stats": "{explorerLink} that modelize the french tax-benefit system.",
+  "explorerText":"Explore the {variablesCount, number} variables and {parametersCount, number} parameters",
+  "api":"{apiLink} that lets you compute them from any software.",
+  "apiText":"Use the API",
```

Comment:
In JSON, the usual convention is to have a space after colons in property name/value pairs, like so:

```
"apiText": "Use the API"
          ^
```

🙂